internal/upload/awsupload/awsupload.go

```diff
@@ -74,7 +74,7 @@ func WaitUntilImportSnapshotTaskCompleted(c *ec2.EC2, input *ec2.DescribeImportS
 func WaitUntilImportSnapshotTaskCompletedWithContext(c *ec2.EC2, ctx aws.Context, input *ec2.DescribeImportSnapshotTasksInput, opts ...request.WaiterOption) error {
 	w := request.Waiter{
 		Name:        "WaitUntilImportSnapshotTaskCompleted",
-		MaxAttempts: 80, // this can take up to 15 minutes, therefore set the timeout to 20 minutes
+		MaxAttempts: 500,
```

Comment:
IMO there isn't much sense to increase this too much. It could fail even with 500, although probably rarely. If the test fails we can always reschedule it or merge with a failing test (not ideal but possible).

Another possibility is to make it easier to change the target region via Jenkins job parameters so we can pick up another region in case of known regional failures/infra issues.

Reply:
Agreed. This was mainly some messing around while I waited for the grub uuid bug to be resolved in #558. 😉